doc/api/stream.md

```diff
@@ -1076,6 +1085,15 @@ also be emitted.
 Calling [`stream.read([size])`][stream-read] after the [`'end'`][] event has
 been emitted will return `null`. No runtime error will be raised.
 
+##### readable.destroyed
+<!-- YAML
+added: REPLACEME
```

Comment:
```suggestion
added: v8.0.0
```

It was added in PR https://github.com/nodejs/node/pull/12925

Reply:
resolved in https://github.com/nodejs/node/pull/28815/commits/5f3113dcf529d0c06c6a8b66a1bde9750b79daed